Binaries/DafnyPrelude.bpl

```diff
@@ -1190,7 +1194,7 @@ axiom (forall<U, V> a: [U] bool, b:[U]V, t:Ty ::
 function Map#Build<U, V>(Map U V, U, V): Map U V;
 /*axiom (forall<U, V> m: Map U V, u: U, v: V ::
   { Map#Domain(Map#Build(m, u, v))[u] } { Map#Elements(Map#Build(m, u, v))[u] }
-  Map#Domain(Map#Build(m, u, v))[u] && Map#Elements(Map#Build(m, u, v))[u] == v);*/
+  Map#Domain(Map#Build(m, u, v))[u] && Map#Elements(Map#Build(m, u, v))[u] == v);
```

Comment:
again, is this removal of end-of-comment intended?

Reply:
Yikes good catch.  I just ran emacs `delete-trailing-whitespace`.  Need to see what happened there.

Reply:
Ug, boogie-mode is broken.  Pushing fix.

Reply:
Thank you, fixed.  That was broken in lots of ways.  I forgot to point the parent to the whitespace change as well, so that whitespace stuff (also fixed) shouldn't have shown up anyway.  Sorry for the noise!